.github/renovate.json5

```diff
@@ -6,5 +6,11 @@
       "matchUpdateTypes": ["minor", "patch", "digest"],
       "automerge": true,
     },
+    {
+      "matchPackageNames": [
+        "ghcr.io/aquasecurity/trivy",
+      ],
+      "semanticCommitType": "fix",
+    },
```

Comment:
How is the logic in Renovate? Could it make sense to add automerge here, or is it covered by the previous section?